pop-up.css

```diff
@@ -0,0 +1,203 @@
+* {
+  margin: 0;
+  padding: 0;
+  outline: 0;
+  border: 0;
+  box-sizing: border-box;
+  text-decoration: none;
+  list-style: none;
+}
+
+html {
+  scroll-behavior: smooth;
+  overflow-x: hidden;
+}
+
+body {
+  background: #eee;
+}
+
+#modal {
+  position: fixed;
+  top: 0;
+  left: 0;
+  min-height: 100vh;
+  width: 100%;
+  background: rgba(0, 0, 0, 0.8);
+  backdrop-filter: blur(4px);
+  display: none;
+  align-items: center;
+  justify-content: center;
+  z-index: 7;
+}
+
+.modal-container {
+  width: 86%;
+  margin: 0 auto;
+}
+
+#modal .popup {
+  padding: 0.5rem;
+  text-align: center;
+  background: #fff;
+  position: relative;
+  width: 80%;
+  overflow-y: auto;
+  height: 90vh;
+  margin: 0 auto 1rem;
+  border-radius: 0.5rem;
+  z-index: 5;
+}
+
+.project-popup .popup.active {
+  display: block;
+}
+
+.header-close {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 0.2rem;
+}
+
+.header-close h3 {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 700;
+  font-size: 1.2rem;
+  color: #174d31;
+  padding-top: 0.4rem;
+}
+
+.card-info {
+  width: 100%;
+}
+
+.main-img {
+  height: auto;
+  width: 74%;
+  margin: 0 auto;
+}
+
```

Comment:
- [ ] Nice work implementing the popup windows :+1: :sunglasses: However the popup images do not follow its content alignment for both the mobile and desktop version (see image). Kindly check it to follow the Figma Design as required.
![Screenshot from 2023-04-19 13-35-32](https://user-images.githubusercontent.com/91142432/233097527-df8ba47b-95b7-4d32-bc01-1f3d05094cff.png)

Reply:
Adjusted container widths to match specification